### ISSUE TITLE
Addon load order flag or load flag.

### ISSUE THREAD

**Issue by Ultimum-Aeternum** [Facepunch/garrysmod-requests#1696]
Load oreder flag:
Ability to put some sort of flag to addon that makes so that it loads before game content is mounted.
It would be very useful if server uses content of any game but player does not have this game and instead has at least some custom low poly/resolution replacements instead of errors. 
Load flag:
Prevent addon from loading if certain game content is present so it does not replace high poly/resolution content.

**Comment by viral32111** [comment]
> Prevent addon from loading if certain game content is present so it does not replace high poly/resolution content.

Can't you do this with [`IsMounted()`](https://wiki.facepunch.com/gmod/Global.IsMounted)?

**Comment by Ultimum-Aeternum** [comment]
> > Prevent addon from loading if certain game content is present so it does not replace high poly/resolution content.
> 
> Can't you do this with [`IsMounted()`](https://wiki.facepunch.com/gmod/Global.IsMounted)?

Ok, lets say I check if game content [`IsMounted()`](https://wiki.facepunch.com/gmod/Global.IsMounted) could you please explain how it will prevent game from replacing models/materials by low poly or what ever custom ones?

**Comment by viral32111** [comment]
Fair enough, I was thinking more from a Lua perspective. Never mind what I said :P
